auth/auth.go

```diff
@@ -93,6 +95,25 @@ func (r *RenewingChecker) IsLoggedIn() bool {
 	return cfg.AutoConnectData.ID != 0 && len(cfg.TokensData) > 0 && isLoggedIn
 }
 
+func (r *RenewingChecker) IsMFAEnabled() (bool, error) {
+	r.mu.Lock()
+	defer r.mu.Unlock()
+
+	var cfg config.Config
+	if err := r.cm.Load(&cfg); err != nil {
+		return false, fmt.Errorf("loading config: %w", err)
+	}
+
+	data := cfg.TokensData[cfg.AutoConnectData.ID]
+
+	resp, err := r.creds.MultifactorAuthStatus(data.Token)
+	if err != nil {
+		return false, fmt.Errorf("querying MFA status: %w", err)
+	}
+
+	return resp.Status == "enabled", nil
```

Comment:
Would be good to have constant defined and used on both ends

Reply:
Ok, here it is from API and might be different thing than in `cli_account.go`

Reply:
But maybe string constants could be defined in `pb` and used consistently instead of having numeric constants in `pb` but converting to/from strings on both ends?